doc/classes/Tween.xml

```diff
@@ -280,6 +280,22 @@
 			<return type="void" />
 			<description>
 				Stops the tweening and resets the [Tween] to its initial state. This will not remove any appended [Tweener]s.
+				[b]Note:[/b] This does [i]not[/i] reset targets of [PropertyTweener]s to their values when the [Tween] first started.
+				[codeblock]
+				var tween := create_tween()
```

Comment:
We don't use static typing in class reference
```suggestion
				var tween = create_tween()
```

Reply:
Applied this suggestion from AThousandShips, thanks!